CMakeLists.txt

```diff
@@ -114,6 +114,12 @@ function(spvtools_default_compile_options TARGET)
     endif()
   endif()
 
+  if (MSVC)
+    # Specify /EHsc for exception handling. This makes using SPIRV-Tools as
+    # dependencies in other projects easier.
+    target_compile_options(${TARGET} PRIVATE /EHsc)
```

Comment:
I think /EHs is better than /EHsc for SPIRV-Tools.  But does /EHs solve your problem?  If so then update the PR.  :-)